src/AutoRest.CSharp/Common/Input/InputTypes/InputModelType.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace AutoRest.CSharp.Common.Input
 {
-    internal record InputModelType(string Name, string? Namespace, string? Accessibility, string? Deprecated, string? Description, InputModelTypeUsage Usage, IReadOnlyList<InputModelProperty> Properties, InputModelType? BaseModel, IReadOnlyList<InputModelType> DerivedModels, string? DiscriminatorValue, string? DiscriminatorPropertyName, InputDictionaryType? InheritedDictionaryType, bool IsNullable)
+    internal record InputModelType(string Name, string? Namespace, string? Accessibility, string? Deprecated, string? Description, InputModelTypeUsage Usage, IReadOnlyList<InputModelProperty> Properties, InputModelType? BaseModel, IReadOnlyList<InputModelType> DerivedModels, string? DiscriminatorValue, string? DiscriminatorPropertyName, InputDictionaryType? InheritedDictionaryType, bool IsNullable, IReadOnlyList<string> MediaTypes)
```

Comment:
why do we need this `MediaTypes` on models?
Is the purpose to know whether this model is directly or indirectly used by a multipart operation?
If so, I remember last time Michael said we could put the information in `usage`.
In my opinion, this is just like "serialization", as we have for now, json and xml.
I am OK with this, but we might need to discuss how we should implement this, especially this is also related with xml

Reply:
Yes. after we adopt tcgc, we will get the `usage`. And current tcgc does not handle multiple-contentType senario ( maybe it will change in future), I think, we may not combine this in our inputModel, just recorded down the MediaType a model support.

Reply:
remove `MediaTypes` and use `Usage`